Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+WORKDIR /code
+ADD . /code
+ENV LINTREVIEW_SETTINGS /code/settings.sample.py
+RUN apt-get update && apt-get install -y php-pear ruby npm && \
```

Comment:
Mostly because they are bookended by the apt-get statements to clean up afterwards and attempt to reduce image bloat.